wds-log-post.php

```diff
@@ -523,11 +532,19 @@ public static function abort_progress( $log_post_id ) {
  * @since  0.1.0
  * @return WDS_Log_Post  Singleton instance of plugin class.
  */
-function wds_log_post() {
-	return WDS_Log_Post::get_instance();
+function wds_log_post( $posttype, $name ) {
+	return WDS_Log_Post::get_instance( $posttype, $name );
 }
 
+add_action( 'plugins_loaded', 'load_wds_log', 9999);
+function load_wds_log() {
 // Kick it off
-if ( apply_filters( 'wds_log_post_site_check', 'is_main_site' ) ) {
-	add_action( 'plugins_loaded', array( wds_log_post(), 'hooks' ) );
+    if ( apply_filters( 'wds_log_post_site_check', 'is_main_site' ) ) {
+        $posttypes = apply_filters( 'wds_log_post_types', array() );
```

Comment:
Ok, I'll mark this for `0.3.0` then, and include a note that the filter will be required to use the plugin.